src/reference/informative/informC08_interpretation_of_mathematics2.rst

```diff
@@ -0,0 +1,216 @@
+.. _informC08_interpretation_of_mathematics2:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Understanding how the mathematics works
+
+    The mathematics of a mathematical model is a collection of statements which are held to be true at all times.
+    These are distinct from initial conditions of a model (specified using the :code:`initial_value` attribute on :code:`variable` elements) which are only true initially.
+    You can read more about initialisation on the :ref:`Interpretation of initial_values<>` page and in the section below this one.
+
+    The collection of mathematical statements in CellML are the set of top-level children of a :code:`math` elements inside "pertinent components" (as explained in the previous block).
```

Comment:
Why "top-level"? Otherwise, I would delete "(as explained in the previous block)" if my above suggestion is to be taken into account.

Reply:
Because maths elements have lots of children?

Reply:
If we think of the following bit of XML:
```xml
<math xmlns="http://www.w3.org/1998/Math/MathML">
    <apply>
        <eq/>
        <ci>g_Na</ci>
        <apply>
            <times/>
            <apply>
                <power/>
                <ci>m</ci>
                <cn cellml:units="dimensionless">3</cn>
            </apply>
            <ci>h</ci>
            <ci>g_Na_max</ci>
        </apply>
    </apply>
    <apply>
        <eq/>
        <ci>i_Na</ci>
        <apply>
            <times/>
            <apply>
                <plus/>
                <ci>g_Na</ci>
                <cn cellml:units="milliS_per_cm2">0.14</cn>
            </apply>
            <apply>
                <minus/>
                <ci>V</ci>
                <ci>E_Na</ci>
            </apply>
        </apply>
    </apply>
</math>
```
the `math` element has two `apply` child elements which correspond to two mathematical statements. Surely, the top-elevel children are those two same `apply` child elements? This is why, I would just say "... are the set of child elements of `math` elements..." (not that I also removed "a" before "`math`"), or maybe "... are the set of `apply` child elements of `math` elements..."?

Reply:
335ff79